packages/core/yarn.lock

```diff
@@ -0,0 +1,2666 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
There shouldn't be a `yarn.lock` file in the core folder as the monorepo uses the one at the root of the project, so this will need to be removed.